catalog/resource_sql_table.go

```diff
@@ -406,6 +430,23 @@ func ResourceSqlTable() common.Resource {
 	return common.Resource{
 		Schema: tableSchema,
 		CustomizeDiff: func(ctx context.Context, d *schema.ResourceDiff) error {
+			if d.HasChange("column") {
```

Comment:
nit: to prevent CustomizeDiff from getting too long, let's split this out into a separate function, esp. given that we will be adding additional verification after this for adding/removing columns.